protos/vega/data_source.go

```diff
@@ -36,6 +36,13 @@ func NewDataSourceSpec(sc *DataSourceDefinition) *DataSourceSpec {
 		case *DataSourceDefinition_Internal:
 			in := sc.GetInternal()
 			if in != nil {
+				switch in.SourceType.(type) {
+				case *DataSourceDefinitionInternal_Time:
+					//t := in.GetTime()
+
+				case *DataSourceDefinitionInternal_TimeTrigger:
+					//t := in.GetTimeTrigger()
```

Comment:
why are those commented out?

Reply:
Because they are not used anywhere yet, this method is created for testing purposes.